rllib/env/single_agent_episode.py

```diff
@@ -383,12 +376,13 @@ def add_env_reset(
 
         self.observations.append(observation)
         self.infos.append(infos)
-        if render_image is not None:
-            self.render_images.append(render_image)
 
         # Validate our data.
         self.validate()
 
+        # Start the timer for this episode.
+        self._start_time = time.time()
```

Comment:
Btw. why do we use `time()` here and not `perf_counter()` which is more precise?

Reply:
done